### ISSUE TITLE
Examples with error. (Failed to convert the response to stream)

### ISSUE THREAD

**Issue by juan-20** [vercel-labs/ai#188]
i dowloaded the next-openai-app and the sveltekit-openai changed the .env.local but i make the request i gives the error bellow;
(next)
```
- error node_modules\ai\dist\index.mjs (112:10) @ AIStream
- error Failed to convert the response to stream. Received status code: 429. 
```
(sveltekit)
```
Error: Failed to convert the response to stream. Received status code: 429. at AIStream 
```
i just did one request and inside open-ai the API is with 0 usage.

this is the sveltekit route "chat":

to clone to project:

 npx create-next-app --example https://github.com/vercel-labs/ai/tree/main/examples/next-openai next-openai-app

**Comment by shuding** [comment]
I think this will be covered by #163. Error code 429 came from OpenAI that you can log out the response body to see more details.

**Comment by eyucherin** [comment]
I am also getting this error for Svelte

**Comment by MaxLeiter** [comment]
Closing as a dupe of https://github.com/vercel-labs/ai/issues/144

**Comment by lgrammel** [comment]
## Identify issue type

Identified as **None**  
Confidence: **High**

### Rationale
This is primarily a support/duplicate report about an upstream OpenAI rate-limit response, not evidence that the SDK or examples are malfunctioning.

### Evidence
- Both example apps receive HTTP 429 responses from OpenAI.
- A maintainer states the 429 originates from OpenAI and recommends inspecting the response body.
- The issue was closed as a duplicate of #144.
